prover/circuits/pi-interconnection/e2e_test.go

```diff
@@ -290,3 +292,22 @@ func decomposeLittleEndian(t *testing.T, digits []int, n, base int) {
 	}
 	assert.Zero(t, n)
 }
+
```

Comment:
This test will be unmaintainable and ressource-consuming for the CI.

It makes sense for debugging the current issues but if we change anything in the pi-interconnection it will stop working and it will be hard to fix. Best to assume we will be undertaking the integration tests from scratch at every release.